<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiagnosisDispansPlanedListDialog</class>
 <widget class="QDialog" name="DiagnosisDispansPlanedListDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Параметры</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,1,1,0">
     <item>
      <widget class="QLabel" name="lblMKB">
       <property name="text">
        <string>Коды диагнозов по &amp;МКБ</string>
       </property>
       <property name="buddy">
        <cstring>cmbMKBFilter</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbMKBFilter">
       <item>
        <property name="text">
         <string>Игнор.</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Интервал</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="CICDCodeEdit" name="edtMKBFrom">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="inputMask">
        <string>a00.00; </string>
       </property>
       <property name="text">
        <string>A.</string>
       </property>
       <property name="maxLength">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CICDCodeEdit" name="edtMKBTo">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="inputMask">
        <string>a00.00; </string>
       </property>
       <property name="text">
        <string>Z99.9</string>
       </property>
       <property name="maxLength">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Врач</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="CPersonComboBoxEx" name="cmbPerson"/>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblBegDate">
       <property name="text">
        <string>с</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CDateEdit" name="edtBegDate"/>
     </item>
     <item>
      <widget class="QLabel" name="lblEndDate">
       <property name="text">
        <string>по</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CDateEdit" name="edtEndDate"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CPersonComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBoxEx.h</header>
  </customwidget>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit</header>
  </customwidget>
  <customwidget>
   <class>CICDCodeEdit</class>
   <extends>QLineEdit</extends>
   <header>library/ICDCodeEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DiagnosisDispansPlanedListDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
